src/Cli/Microsoft.DotNet.Cli.Utils/NuGetPackageInstaller.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    internal class NuGetPackageInstaller
+    {
+        private static readonly string sourceUrl = "https://api.nuget.org/v3/index.json";
```

Comment:
Should this the dotnet-public feed instead: https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json?

Reply:
This will come later. We want to add this to unblock other thread first